### PR TITLE
fixing layout issue for code block

### PR DESCRIPTION
**Pull request by l0k0ms**
To address this issue https://github.com/DataDog/documentation/issues/1429

### CODE REVIEW

File: content/integrations/ecs.md [hunk 1]
```diff
@@ -95,41 +96,36 @@ Ideally you want the Datadog agent to load on one container on each EC2 instance
...
-    This user script above will:
-
-    * Start the task defined with the right parameters
-    * Add a few lines to `/etc/rc.local` so that the rebooted instance starts the task
+2. Create a new instance by clicking the **Launch Instance** button.
+3. Click on Community AMIs. Visit [this page to see a list of current ECS optimized instances](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_container_instance.html). Choose the appropriate AMI for your region and copy the ID into the search box. Choose the AMI that comes up as a result of the  search.
+4. Follow the prompts as you normally would when setting up an instance.
+5. On the third dialog, select the IAM role you created above.
+6. Expand the Advanced Details section and copy the following script into the User Data section. Change cluster name to your cluster's name and task definition to the name you gave your task definition.
+{{< highlight bash >}}
```
Comment (kmshultz) [inline content/integrations/ecs.md]: Please add a blank line before this one.

File: content/integrations/ecs.md [hunk 2]
```diff
@@ -95,41 +96,36 @@ Ideally you want the Datadog agent to load on one container on each EC2 instance
...
+  * Start the task defined with the right parameters
+  * Add a few lines to `/etc/rc.local` so that the rebooted instance starts the task
 
 #### Create a new CoreOS instance
 
 1. Log in to the AWS console and navigate to the EC2 section.
-1. Create a new instance  as described in the instructions for a simple CoreOS ECS instance [here](https://coreos.com/os/docs/latest/booting-on-ecs.html)).
-1. When you get to the Configure Instance Details step, select the IAM role you created above.
-1. Paste the following block in User Data under Advanced Details, replace `CLUSTER_NAME` and `YOUR_API_KEY` with the ECS cluster that instance will join and your Datadog API key. This block declares two units with cloud-config, one for the ecs-agent container, used by Amazon ECS to administrate the ECS instance, and one for the dd-agent container, used by Datadog to collect metrics about the system and the tasks running on this ECS instance. Of course it can be modified to include your own tasks as well.
+2.  Create a new instance  as described in the instructions for a simple CoreOS ECS instance [here](https://coreos.com/os/docs/latest/booting-on-ecs.html)).
```
Comment (kmshultz) [inline content/integrations/ecs.md]: One too many spaces between "2" and "Create". Same with the next few steps.

File: content/integrations/ecs.md [hunk 3]
```diff
@@ -95,41 +96,36 @@ Ideally you want the Datadog agent to load on one container on each EC2 instance
...
+3.   When you get to the Configure Instance Details step, select the IAM role you created above.
+4.   Paste the following block in User Data under Advanced Details, replace `CLUSTER_NAME` and `YOUR_API_KEY` with the ECS cluster that instance will join and your Datadog API key.
```
Comment (kmshultz) [inline content/integrations/ecs.md]: s/replace/replacing

File: content/integrations/ecs.md [hunk 4]
```diff
@@ -95,41 +96,36 @@ Ideally you want the Datadog agent to load on one container on each EC2 instance
...
+2.  Create a new instance  as described in the instructions for a simple CoreOS ECS instance [here](https://coreos.com/os/docs/latest/booting-on-ecs.html)).
+3.   When you get to the Configure Instance Details step, select the IAM role you created above.
+4.   Paste the following block in User Data under Advanced Details, replace `CLUSTER_NAME` and `YOUR_API_KEY` with the ECS cluster that instance will join and your Datadog API key. 
+This block declares two units with cloud-config, one for the ecs-agent container, used by Amazon ECS to administrate the ECS instance, and one for the dd-agent container, used by Datadog to collect metrics about the system and the tasks running on this ECS instance. Of course it can be modified to include your own tasks as well.
 
 
        #cloud-config
```
Comment (kmshultz) [inline content/integrations/ecs.md]: Do we want a {{highlight}} block here, too?

File: content/integrations/ecs.md (hunk 1 above)
Comment (l0k0ms) [inline content/integrations/ecs.md]: I can't otherwise it breaks the bullet points layout.

File: content/integrations/ecs.md (hunk 4 above)
Comment (l0k0ms) [inline content/integrations/ecs.md]: 👍

File: content/integrations/ecs.md (hunk 3 above)
Comment (l0k0ms) [inline content/integrations/ecs.md]: 👍

File: content/integrations/ecs.md (hunk 2 above)
Comment (l0k0ms) [inline content/integrations/ecs.md]: 👍

File: content/integrations/ecs.md [hunk 5]
```diff
@@ -95,86 +94,80 @@ Ideally you want the Datadog agent to load on one container on each EC2 instance
...
+
+         [Service]
+         Environment=API_KEY=YOUR_API_KEY
+         Environment=TAGS=simple-tag-0
+         ExecStartPre=/usr/bin/docker pull datadog/docker-dd-agent:latest
+         ExecStart=/usr/bin/docker run --name dd-agent --env=API_KEY=${API_KEY} --volume=/var/run/docker.sock:/var/run/docker.sock --volume=/proc/:/host/proc/:ro --volume=/sys/fs/cgroup/:/host/sys/fs/cgroup:ro datadog/docker-dd-agent:latest
+         ExecStop=/usr/bin/docker stop dd-agent
+{{< /highlight >}}
 
 ### That's all!
```
Comment (kmshultz) [inline content/integrations/ecs.md]: This is cheesy. Let's remove this header.

**Pull request merged**